src/main/java/com/google/opensesame/servlets/PersonBuilder.java

```diff
@@ -38,34 +41,55 @@ public PersonBuilder interestTags(ArrayList<String> interestTags) {
     return this;
   }
 
-  public PersonBuilder projectIDs(ArrayList<String> projectIDs) {
+  public PersonBuilder menteeIDs(ArrayList<String> interestTags) {
+    this.interestTags = interestTags;
+    return this;
+  }
+
+  public PersonBuilder projectIDs(ArrayList<Long> projectIDs) {
     this.projectIDs = projectIDs;
     return this;
   }
 
+  public PersonBuilder email(String email) {
+    this.email = email;
+    return this;
+  }
+
+  public PersonBuilder userID(String userID) {
+    this.userID = userID;
+    return this;
+  }
+
   /**
    * Create a PersonObject instance with information from the current builder.
    *
    * @return PersonObject that corresponds with feilkds from this PersonBuilder instance
    * @throws IOException
    */
-  public PersonObject buildPersonObject() throws IOException {
-    return new PersonObject(gitHubID, interestTags);
+  public PersonObject buildPerson() throws IOException {
+    return new PersonObject(userID, gitHubID, interestTags, email);
   }
 
   /**
-   * Convert an entity retrieved from Datastore into a PersonObject.
+   * Fill PersonBuilder information from an entity. This will work on PersonEntities and
+   * MentorEntities
    *
-   * @param personEntity PersonObject.ENTITY_NAME entity
+   * @param personEntity PersonEntity instance with user infromation
    * @return PersonObject that corresponds to the entity retrieved from datastore
    * @throws IOException
    */
-  public PersonObject buildPersonObject(Entity personEntity) throws IOException {
-    String entityGitHubID = (String) personEntity.getProperty(GITHUB_ID_FIELD);
-    this.gitHubID(entityGitHubID);
-    ArrayList<String> entityTagList = (ArrayList<String>) personEntity.getProperty(TAG_LIST_FIELD);
-    this.interestTags(entityTagList);
-    return this.buildPersonObject();
+  public PersonBuilder fromEntity(PersonEntity personEntity) throws IOException {
```

Comment:
Nice! I'm loving the naming consistency with this and the projects stuff. Will definitely help to keep things readable!